airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py

```diff
@@ -123,6 +123,52 @@ def primary_keys_for_records(streams, records):
 
 @pytest.mark.default_timeout(5 * 60)
 class TestBasicRead(BaseTest):
+    @staticmethod
+    def _validate_schema(records, configured_catalog):
```

Comment:
unit tests?

Reply:
unfortunately not for the whole test, but we have a test for validate_schema